src/test/java/com/TableTalk/Enterprise/EnterpriseApplicationTests.java

```diff
@@ -6,5 +6,10 @@
 @SpringBootTest
 class EnterpriseApplicationTests {
 
+    @Test
+    void contextLoads() {
+    }
+
+    // TODO Create unit tests to reflect requirements and functionality
 
```

Comment:
As an empty class, this isn't adding value.  Put something in it, or don't change/add it at all.